controller_manager:
  ros__parameters:
    update_rate: 100 

rack_pinion_controller:
  ros__parameters:
    type: ackermann_steering_controller/RackPinionController
    front_steer_joints:
      - rack_pinion_joint
    wheel_radius: 0.3
    track_width: 1.2
    radius_rotation: 1.0
    front_wheel_track: 1.2
    rear_wheel_track: 1.2
    wheelbase: 1.5
